doc/changes.md

```diff
@@ -1,5 +1,70 @@
+* Improve the docstring of the pygmt package ([#1016](https://github.com/GenericMappingTools/pygmt/pull/1016))
+* Add gallery example showing different polar projection use cases ([#955](https://github.com/GenericMappingTools/pygmt/pull/955))
+* Add common alias coltypes (-f) for specifying i/o data types ([#994](https://github.com/GenericMappingTools/pygmt/pull/994))
+* Add a gallery example for plotting Cartesian, circular and geographic vectors ([#950](https://github.com/GenericMappingTools/pygmt/pull/950))
+* Expand documentation linking in CONTRIBUTING.md ([#802](https://github.com/GenericMappingTools/pygmt/pull/802))
+* Write changelog in markdown using MyST ([#941](https://github.com/GenericMappingTools/pygmt/pull/941))
+* Update web font to Atkinson Hyperlegible ([#938](https://github.com/GenericMappingTools/pygmt/pull/938))
+* Add a gallery example showing different vector heads and tails ([#890](https://github.com/GenericMappingTools/pygmt/pull/890))
+* Improve the gallery example for datetime inputs ([#919](https://github.com/GenericMappingTools/pygmt/pull/919))
+
+### Maintenance
+
+* Refactor plot and plot3d to use virtualfile_from_data ([#990](https://github.com/GenericMappingTools/pygmt/pull/990))
+* Explicitly exclude unnecessary files in source distributions ([#999](https://github.com/GenericMappingTools/pygmt/pull/999))
+* Refactor grd modules to use virtualfile_from_data ([#992](https://github.com/GenericMappingTools/pygmt/pull/992))
+* Refactor info and grdinfo to use virtualfile_from_data ([#961](https://github.com/GenericMappingTools/pygmt/pull/961))
+* Onboarding maintainer checklist ([#773](https://github.com/GenericMappingTools/pygmt/pull/773))
+* Add comprehensive tests for pygmt.clib.loading.clib_full_names ([#872](https://github.com/GenericMappingTools/pygmt/pull/872))
+* Add a workflow checking links in plaintext and HTML files ([#634](https://github.com/GenericMappingTools/pygmt/pull/634))
+* Remove nbsphinx extension ([#931](https://github.com/GenericMappingTools/pygmt/pull/931))
+* Improve the error message for loading an old version of the GMT library ([#925](https://github.com/GenericMappingTools/pygmt/pull/925))
+* Move requirements-dev.txt dependencies to environment.yml ([#812](https://github.com/GenericMappingTools/pygmt/pull/812))
+* Ensure proper non-dev version string when publishing to PyPI ([#900](https://github.com/GenericMappingTools/pygmt/pull/900))
+* Run tests in a single CI job (Ubuntu + Python 3.9) for draft PRs ([#906](https://github.com/GenericMappingTools/pygmt/pull/906))
+
+### Contributors
+
+* [Dongdong Tian](https://github.com/seisman)
+* [Wei Ji Leong](https://github.com/weiji14)
+* [Jiayuan Yao](https://github.com/core-man) 
+* [Meghan Jones](https://github.com/meghanrjones)
+* [Michael Grund](https://github.com/michaelgrund)
+* [Will Schlitzer](https://github.com/willschlitzer)
+* [Liam Toney](https://github.com/liamtoney)
+* [Kathryn Materna](https://github.com/kmaterna)
```

Comment:
There are a few contributors that are not included here, why?

Reply:
I included everyone in AUTHORS.md; should I have included those who haven't added themselves?

Reply:
Yes, please include everyone. There should be a git log command to check who commited between v0.3.0 and v0.3.1

Reply:
See here. We want to include every one. https://github.com/GenericMappingTools/pygmt/blob/master/AUTHORSHIP.md#changelog-for-each-release

Reply:
I added @tawandamoyo and @aliciaha1997, which I think completes the list. Both @michaelgrund and @core-man both look like they had repeat entries when I ran `git shortlog HEAD...v0.3.0 -sne`, and I figured I wouldn't include dependabot (this will definitely be a problem for me if and when there is a robot uprising).

Reply:
> I figured I wouldn't include dependabot (this will definitely be a problem for me if and when there is a robot uprising).

There should a way to prevent the bot uprising soon with https://github.com/release-drafter/release-drafter/issues/569 :slightly_smiling_face: